parsec/core/fs/userfs/userfs.py

```diff
@@ -1057,11 +1057,7 @@ async def workspace_start_reencryption(self, workspace_id: EntryID) -> Reencrypt
                 workspace_entry.id, new_workspace_entry.encryption_revision, now, reencryption_msgs
             )
             if not ok:
-                # Participant list has changed concurrently
-                logger.info(
```

Comment:
was this log unnecessary ?

Reply:
yup, wasn't worth mentioning to the user compared all the concurrency handling that is done in workspace fs transaction module

![image](https://user-images.githubusercontent.com/3187637/131702064-c44bb628-157d-4088-8a69-6b801c653309.png)